setup.py

```diff
@@ -2,9 +2,8 @@
 import os
 import re
 import datetime
-
+from codecs import open     #pylint: disable=redefined-builtin
```

Comment:
Is there a reason we don't just import this as a different name and then do that? E.g. call it codec_open or something?

Reply:
Upon revisiting, we might be able to remove this altogether. I'll test this first and send out a commit if it works.

Reply:
After testing this with the native `open` module, we may want to leave this as is. By importing `codecs`, we maintain consistency between Python 2 and 3. This is because the `open` module doesn't support encoding type in Python 2, however, `codecs` does. We need to set encoding type because Mac and Windows use different ones.

@kevcunnane I like this method because it overrides the native `open` module, which ensures we'll use the `codecs` version. What are your thoughts?